sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -892,16 +888,15 @@
         ],
         "responses": {
           "200": {
-            "description": "Returns the module twin object",
+            "description": "Returns the module twin object.",
             "schema": {
               "$ref": "#/definitions/Twin"
             }
           }
         }
       },
       "put": {
-        "summary": "Replaces tags and desired properties of a module twin.",
-        "description": "Replaces a module twin. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "description": "Replaces the tags and desired properties of a module. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information.",
```

Comment:
I think we should all think about a consistent term for updates in general. Either we always say "Update" or "Replace". And the text should read "Updates/Replaces a module twin". From what I've seen so far, I think we're using update.

Reply:
Also this is PUT, so why are you not saying Create or Update?

Reply:
I believe we've discussed and resolved this point. Let me know if you still have questions @vinagesh !

Reply:
Is there no CREATE on PUT for modules also like twins?

Reply:
Yes, correct.